src/test/java/io/ebean/annotation/ForPlatform.java

```diff
@@ -0,0 +1,17 @@
+package io.ebean.annotation;
```

Comment:
Maybe you can take a look at this. I see that you change lots of package-names to be java9 compatible. I don't know if duplicate packages causes issues in unittests (there's no need to move this annotation to the ebean-annotations artifact IMHO)

Reply:
> no need to move this annotation to the ebean-annotations

I agree. LGTM